### ISSUE TITLE
Course duplication job fails if images cannot be downloaded

### ISSUE THREAD

**Issue by trangmei** [Coursemology/coursemology2#3485]
![image](https://user-images.githubusercontent.com/35135264/56632117-d9f8ec00-668a-11e9-9942-20900b8d6c15.png)

**Comment by trangmei** [comment]
- [ ] Add rake task to find all files that would result in 404 on `download!`. We'll decide what to do with them later.
- [ ] Parse job error and pass relevant info to UI to aid debugging in the future.

**Comment by AndreiUngur** [comment]
Hi,

I'm not sure how to go about reproducing this issue. Here's what I've tried:
- Create a course
- Under the `Materials` tab, I upload an image
- In my file explorer, I go to `public/uploads/attachments` and find my uploaded image
- I rename the uploaded image to something else (or delete it)
- Open a rails console & run:
```
ActsAsTenant.current_tenant = Instance.first
Course::DuplicationJob.perform_now(Course.first)
```
which executes correctly with no errors, even if the uploaded image is presumably not found.

I've also tried changing the name of non-uploaded files (such as those under `public/img`) but still had no problem running the duplication job.
